config.ini_sample

```diff
@@ -73,10 +73,17 @@ disable_mempool_fee_histogram = false
 # Parameter for broadcasting unconfirmed transactions
 # Options are:
 # * own-node         (broadcast using the connected full node)
+# * tor              (broadcast to random nodes over tor)
 # * system <cmd> %s  (save transaction to file, and invoke system command
 #                     with file path as parameter %s)
 broadcast_method = own-node
 
+# For tor broadcasting, comment out the above line, and uncomment the one below
```

Comment:
Tell people to just edit the option rather than commenting out one and uncommenting the other. Maybe use `For tor broadcasting (broadcast_method = tor) configure the tor proxy host and port below`. I'd leave out the point about Core 0.18 because eventually it will become out of date when we get to 0.19 and 0.20, that comment belongs in an error message and release notes.